### PR TITLE
Moban repo support

### PR DESCRIPTION
**Pull request by jayvdb**
**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `corobo mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: .moban.yaml [hunk 1]
```diff
@@ -43,13 +43,15 @@ dependencies:
...
 
+requires:
+  - https://gitlab.com/coala/mobans.git
 configuration:
   template_dir:
     - .moban.dt/
-    - ../coala-mobans/templates/
-    - ../coala-mobans/assets/
+    - mobans:templates/
+    - mobans:assets/
```
Comment (PrajwalM2212) [inline .moban.yaml]: Needs to be in quotes.
Forward slash may be removed.

Comment (jayvdb) [reply .moban.yaml]: I doesnt need to be in quotes (otherwise the mobans job would fail), and it is more clear what it is when it has a trailing `/`

Comment (PrajwalM2212) [reply .moban.yaml]: 👍 I was referring to https://github.com/moremoban/moban/tree/master/docs/level-10-moban-dependency-as-git-repo

**Comment by jayvdb** [comment]
ack https://github.com/coala/coala/pull/5893/commits/d02449618017d02ec04bd631a79b819a26e40d31 https://github.com/coala/coala/pull/5893/commits/0636e9b44579bd8a6638b1533fef46f106e5b63a

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**